webpack/assets/javascripts/react_app/components/HostsIndex/Columns/core.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { translate as __ } from '../../../common/I18n';
+import RelativeDateTime from '../../common/dates/RelativeDateTime';
+import forceSingleton from '../../../common/forceSingleton';
+
+const coreHostsIndexColumns = [
+  {
+    columnName: 'name',
+    title: __('Name'),
+    wrapper: ({ name }) => <Link to={`hosts/${name}`}>{name}</Link>,
```

Comment:
Neat idea. I like it. However there are a quite a few pages hard coding the url's instead of relying on the API. Even if we decide that is bad and needs a change we'd have to address them en masse . May be we can file a redmine for that and track it as a separate issue

Reply:
> I know today it's common to link to names, but I think it's fragile and we should link to IDs. Hostnames can change, IDs can't.

True, but it's also nice to see your hostname in the url bar when you're on the page. When we originally added this page it was linking to, say, `new/hosts/42` and that was a much worse experience IMO.